packages/maskbook/src/extension/options-page/DashboardComponents/TransferDialog.tsx

```diff
@@ -0,0 +1,284 @@
+                helperText={t('wallet_transfer_balance', {
+                    balance: formatBalance(new BigNumber(tokenBalance ?? '0'), token.decimals ?? 0),
+                    symbol: token.symbol,
+                })}
+            />
+            <TextField
+                required
+                label={t('wallet_transfer_to_address')}
+                placeholder={t('wallet_transfer_to_address')}
+                value={address}
+                onChange={(ev) => setAddress(ev.target.value)}
+            />
+            <TextField
+                label={t('wallet_transfer_memo')}
+                placeholder={t('wallet_transfer_memo_placeholder')}
+                value={memo}
+                disabled={token.type === EthereumTokenType.ERC20}
+                onChange={(ev) => setMemo(ev.target.value)}
+            />
+            <Button
+                className={classes.button}
+                variant="contained"
+                color="primary"
+                disabled={!!validationMessage || transferState.type === TransactionStateType.WAIT_FOR_CONFIRMING}
+                onClick={onTransfer}>
+                {validationMessage || t('wallet_transfer_send')}
+            </Button>
+        </div>
+    )
+}
+//#endregion
+
+//#region receive tab
+const useReceiveTab = makeStyles((theme: Theme) =>
+    createStyles({
+        qr: {
+            marginTop: theme.spacing(2),
+        },
+        form: {
+            padding: theme.spacing(1),
+        },
+    }),
+)
+
+interface ReceiveTabProps {
+    wallet: WalletRecord
+    onClose: () => void
+}
+
+function ReceiveTab(props: ReceiveTabProps) {
+    const { wallet } = props
+    const { t } = useI18N()
+    const classes = useReceiveTab()
+
+    const [, copyToClipboard] = useCopyToClipboard()
+    const copyWalletAddress = useSnackbarCallback(async (address: string) => copyToClipboard(address), [])
+    return (
+        <>
+            <form className={classes.form}>
+                <TextField
+                    required
+                    label={t('wallet_address')}
+                    value={wallet.address}
+                    InputProps={{
+                        endAdornment: (
+                            <InputAdornment position="end">
+                                <IconButton
+                                    size="small"
+                                    onClick={(e: React.MouseEvent<HTMLButtonElement>) => {
+                                        e.stopPropagation()
+                                        copyWalletAddress(wallet.address)
+                                    }}>
+                                    <FileCopyOutlinedIcon />
+                                </IconButton>
+                            </InputAdornment>
+                        ),
+                    }}
+                    variant="outlined"
+                />
+            </form>
+            <Box
+                className={classes.qr}
+                sx={{
```

Comment:
I strongly not suggest you mix using `makeStyles` and `Box`. MUI is migrating its styling engine to emotion. Now `Box` is powered by emotion and `makeStyles` is powered by JSS. If you mix them and you need to overwrite some property, you may encounter CSS order problem

Reply:
@lelenei Take a look.

Reply:
ok